presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveMetadata.java

```diff
@@ -371,6 +401,147 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
         return TableStatisticsMaker.getTableStatistics(typeManager, constraint, handle, icebergTable);
     }
 
+    public void createView(ConnectorSession session, ConnectorTableMetadata viewMetadata, String viewData, boolean replace)
+    {
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
+                .put(TABLE_COMMENT, "Presto View")
```

Comment:
Nit: We can add these constant in MetastoreUtil

Reply:
Done.